cylc/flow/network/schema.py

```diff
@@ -1246,9 +1259,21 @@ class Meta:
 # Generic containers
 class GenericResponse(ObjectType):
     class Meta:
-        description = """Container for command queued response"""
+        description = """Container for workflow command queued response"""
+
+    workflowId = String()
+    success = Boolean(required=True)
+    message = String(required=True)
+
 
-    result = GenericScalar()
+class GenericResponseTuple(NamedTuple):
```

Comment:
You can just use above Graphene `ObjectType`, `GenericResponse`, as a data container.. Why do we need this one?

Reply:
> So looks good.. Few minor comments below..
> 
> The main issue I'm having (when trying to run this and the UIS sibling), is getting it working

Sorry, should have mentioned: It's not working from the UI yet as some changes need to be made there. But GraphiQL is working.